monitoring_service/test/test_e2e.py

```diff
@@ -106,29 +106,29 @@ def test_e2e(
     )
 
     monitoring_service.transport.receive_fake_data(monitor_request.serialize_full())
-    gevent.sleep(0)
+    gevent.sleep(0.01)
     assert channel_id in monitoring_service.monitor_requests
 
     c2.close_channel(c1.address, balance_proof_c1)
-    # wait till monitor's blockchain confirms closeChannel event...
-    wait_for_blocks(10)
-    gevent.sleep(1)
-    # wait till validator confirms NewBalanceProofReceived event...
-    wait_for_blocks(10)
-    gevent.sleep(1)
+    # Wait one block until the ChannelClosed event is confirmed and handled
+    # by the MS
+    wait_for_blocks(1)
+    # Now give the monitoring service a chance to submit the missing BP
+    gevent.sleep(0.01)
     assert [e.event for e in blockchain_validator.events] == ['NewBalanceProofReceived']
 
+    # wait for settle timeout
+    wait_for_blocks(15)
     c2.settle_channel(
         c1.address,
         (balance_proof_c2.transferred_amount, balance_proof_c1.transferred_amount),
         (balance_proof_c2.locked_amount, balance_proof_c1.locked_amount),
         (balance_proof_c1.locksroot, balance_proof_c1.locksroot)
     )
-    wait_for_blocks(10)
-    gevent.sleep(1)
-    # channel is settled
+    # Wait until the ChannelSettled is confirmed
     wait_for_blocks(1)
-    gevent.sleep(1)
+    # Let the MS claim its reward
+    gevent.sleep(0.1)
```

Comment:
same as above

Reply:
In this case it takes longer than `TEST_POLL_INTERVAL`. I don't know why, but I'll keep it like this for now to work on more important tasks.